src/encoded/schemas/experiment_set.json

```diff
@@ -97,6 +97,13 @@
             "lookup": 20,
             "formInput": "textarea"
         },
+        "disease_name": {
```

Comment:
Minor suggestion to call this disease_term - helps convey it is a link to an OntologyTerm.